docs/installation/upgrade/include/2.4-acls.inc

```diff
@@ -0,0 +1,101 @@
+### ACLs
+
+#### Use Global ACL Files instead of Global ACL Directories
+
+To migrate the ACL directories into their respective files you have to do the
+following:
+
+1. Create a new consolidated [[link,acl_global_file]].
+2. For each subdirectory in the currently configured ACL directory, add a line
+   starting with the mailbox name followed by the appropriate content.
+3. Change the `vfile` parameter to the new ACL file.
+4. Remove the old ACL directory parent.
+
+##### Example
```

Comment:
This example contradicts the above text. The text is about converting acl directories to a global acl file. This is also needed for v3.0. The example is about converting acl directories to v3.1 acl settings in dovecot.conf.